cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -251,26 +250,27 @@ func (p *Planner) categorizeNodes(podDestinations map[string]bool, scaleDownCand
 	}
 	p.nodeUtilizationMap = utilizationMap
 	timer := time.NewTimer(p.context.ScaleDownSimulationTimeout)
+RemovalSimulation:
```

Comment:
nit, optional: I'd consider extracting an extra function (e.g. `timedOut(time.Timer) bool`) instead of introducing the loop label. I'm a fan of short functions :)

Reply:
Not sure I understand your proposal. Do you mean extract the whole loop to another function or perform a for loop with additional condition `timedOut(time.Timer)`?

Reply:
Extracting the whole loop into the function will reduce readability as for me, because the function will have a lot of parameters.

Reply:
Yeah, extracting the loop might not be very readable, I was thinking about replacing the whole select statement with:

```
if timedOut(timer) {
    break
}
```

to avoid the need for a label.